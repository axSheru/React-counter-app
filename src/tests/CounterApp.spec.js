import { shallow } from "enzyme";
import CounterApp from "../CounterApp";

describe('Pruebas en CounterApp component.', () => {

    test('debe de hacer match con el snapshot.', () => {

        const wrapper = shallow( <CounterApp /> );

        expect( wrapper ).toMatchSnapshot();

    });

    test('debe de mostrar el valor de 100.', () => {

        const wrapper = shallow( <CounterApp value={ 100 } /> );

        const valor = wrapper.find( 'h2' ).text();

        expect( valor ).toContain( '100' );

    });

});