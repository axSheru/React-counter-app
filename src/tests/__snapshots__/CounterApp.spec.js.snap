// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en CounterApp component. debe de hacer match con el snapshot. 1`] = `
<Fragment>
  <h1>
    CounterApp
  </h1>
  <h2>
    12
  </h2>
  <button
    onClick={[Function]}
  >
    +1
  </button>
  <button
    onClick={[Function]}
  >
    Reset
  </button>
  <button
    onClick={[Function]}
  >
    -1
  </button>
</Fragment>
`;
